tests/test_database.py

```diff
@@ -687,6 +688,38 @@ def test_update(tmpdir):
 
     assert db.update(db) == db
 
+    # databases with schemes that are not compatible
+
+    other_bad = audformat.testing.create_db(minimal=True)
+    other_bad.schemes['labels'] = audformat.Scheme(labels=[1, 2, 3])
+    with pytest.raises(TypeError):
```

Comment:
Currently a `TypeError` is raised if the dtype of the labels do not match. I suggest we change it to a `ValueError` in a separate PR.

Reply:
I created https://github.com/audeering/audformat/issues/277